docs/user/advanced_features.rst

```diff
@@ -187,6 +211,20 @@ must be defined in the WEC-Sim input file, for example:
 
 .. _user-advanced-features-time-step:
 
+
+FIR Filters
```

Comment:
@salhus Minor comment can we spell out FIR and say "Finite Impulse Response (FIR)". Also on line 218 below write out fully the first time and then use FIR for the rest.

Reply:
Makes sense.
- done.